000-local-env.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
```

Comment:
I think you meant to use the `WPCOM_IS_VIP_ENV` constant? `VIP_GO_ENV` tracks the environment name of the site - production/staging/development. That could (and probably should) exist on local environments